### ISSUE TITLE
Missing Accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#40]
Hello! I noticed that the study group map might be missing some accessibility features. It would be great if we could make it more accessible for everyone. Thanks!
